src/model.py

```diff
@@ -151,6 +153,11 @@ def init_engine(self):
             AsyncEngineArgs(**self.vllm_engine_config)
         )
 
+        # Create vLLM custom Metrics
+        labels = {"model": "vllm_metrics", "version": "1"}
```

Comment:
I don't think this should be hard-coded, it should match the Triton model name/version:
```python
labels = {"model": self.args["model_name"], "version": self.args["model_version"]}
```
```suggestion
        labels = {"model": self.args["model_name"], "version": self.args["model_version"]}
```

Reply:
Fixed